### PR TITLE
READY: UTF-8 config

### PR DESCRIPTION
**Pull request by xoriole**

### CODE REVIEW

File: TriblerGUI/tribler_request_manager.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def on_finished(self, reply, capture_errors):
 
         data = reply.readAll()
         try:
-            json_result = json.loads(str(data), encoding='latin_1')
+            json_result = json.loads(str(data), encoding='utf-8')
```
Comment (qstokkink) [inline TriblerGUI/tribler_request_manager.py]: We also send non-utf-8 encoded strings over the request manager.

Comment (xoriole) [reply TriblerGUI/tribler_request_manager.py]: Request manager still uses `latin_1` now and the config is converted to use `UTF-8`.

**Comment by devos50** [comment]
@xoriole is this one ready for merge/review?

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by qstokkink** [comment]
As I do not believe this PR will solve our problems long term and it has gone stale, I will close this PR. It should be naturally fixed when we move to Python 3 anyway.

**Pull request closed without merging**